AnnoyancesFilter/sections/widgets.txt

```diff
@@ -36,6 +36,7 @@ mhealth.ru##.incut_readmore
 !
 ||widget.info-static.ru/js/init.js
 ||ladesk.com/scripts/track.js
+||byjus.com/byjusweb/js/tracking-2.min.js
```

Comment:
Looks like this rule blocks also chat widget, please add a separate rule for the widget to the annoyances filter